include/solution_transfer/meshfree_interpolation.h

```diff
@@ -330,13 +332,17 @@ class InverseDistanceInterpolation : public MeshfreeInterpolation
    */
   InverseDistanceInterpolation (const libMesh::Parallel::Communicator & comm_in,
                                 const unsigned int n_interp_pts = 8,
-                                const Real  power               = 2) :
+                                const Real  power               = 2,
+                                const Number background_value   = 0.,
```

Comment:
We also need more documentation here, for the benefit of users who aren't using it indirectly via a triangulator.